pkgs/development/python-modules/picobox/default.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, flask
+, isPy27
+, pytestCheckHook
+, pythonAtLeast
+, setuptools-scm
+}:
+
+buildPythonPackage rec {
+  pname = "picobox";
+  version = "2.2.0";
+
+  # currently broken on 3.10 due to https://github.com/ikalnytskyi/picobox/issues/55
+  disabled = isPy27 || pythonAtLeast "3.10";
```

```suggestion
  disabled = pythonOlder "3.7";
```
We can safely ignore those tests, the output did just change.

---

I initially packaged this with tests, and then saw the some tests are broken, on Python 3.10, while they succeed on 3.9.

I'm not sure if openapi or other potential downstream consumers of this package make use of the (currently broken) functionality that picobox exposes, and rather mark it as broken for 3.10.

---

As written further down, tests for python 3.10 are fixed by `fetchpatch`'ing another commit (not yet released). This is now not marked as broken for 3.10 anymore.